timemachine/cpp/src/kernels/k_periodic_torsion.cuh

```diff
@@ -104,7 +101,7 @@ void __global__ k_periodic_torsion(
     RealType phase = params[phase_idx];
     RealType period = params[period_idx];
 
-    RealType prefactor = kt * sin(period * angle - phase) * period;
+    RealType prefactor = -kt * sin(period * angle - phase) * period;
```

Comment:
To be clear, the only significant change to this kernel is the addition of the `-` to the prefactor?

Reply:
Nevermind, missed the reversal of the summations of r0, r1, etc.